src/pages/communication-page/components/postal-address/postal-address.scss

```diff
@@ -0,0 +1,8 @@
+.displayAddress {
```

Comment:
I think it's nice to use the convention that we use in the `.scss` for other components that the top-level selector is the component directive selector.

Reply:
fixed